backend/src/openarchiefbeheer/destruction/models.py

```diff
@@ -229,6 +229,9 @@ def abort_destruction(self) -> None:
         self.set_status(ListStatus.new)
         self.planned_destruction_date = None
         self.processing_status = InternalStatus.new
+
+        for assignee in self.assignees.filter(role=ListRole.archivist):
```

Comment:
I went a little protective here as AFAIK there is no database constraint preventing multiple assignees with role archivist from being set.